src/lib/components/artwork_grids/infinite_scroll_grid.tsx

```diff
@@ -35,52 +28,56 @@ export const PageEndThreshold = 1000
  *   - the calculation currently only takes into account the size of the image, not if e.g. the sale message is present
  */
 
-type Artworks = Array<{
-  __id: string,
-  image: {
-    aspect_ratio: number | null,
-  } | null,
-}>
+type Artworks = Array<
+  {
+    __id: string,
+    image:
+      | {
+        aspect_ratio: number | null,
+      }
+      | null,
+  }
+>
 
 interface Props extends ArtistRelayProps, GeneRelayProps {
   /** The direction for the grid, currently only 'column' is supported . */
-  sectionDirection: string;
```

Comment:
Heh - yep.

I took a quick look into this, in running for this PR, I set it to `--trailing-comma es5` which _could_ be inferred to mean don't do trailing commas for an interface, however:

```sh
 !  ~/d/p/a/i/emission   consignments *$  yarn prettier -- --parser typescript src/lib/app_registry.tsx --no-semi --trailing-comma all --write --print-width 120
yarn prettier v0.24.4
$ "/Users/orta/dev/projects/artsy/ios/emission/node_modules/.bin/prettier" --parser typescript src/lib/app_registry.tsx --no-semi --trailing-comma all --write --print-width 120
```

Gives

```diff
 interface Props extends ViewProperties {
-  trigger1pxScrollHack?: boolean,
+  trigger1pxScrollHack?: boolean
 }
```

So taking the comma option into account could make sense. However, for context, Microsoft's [example code](https://www.typescriptlang.org/docs/handbook/interfaces.html) for interfaces uses semis:

```
interface SquareConfig {
    color?: string;
    width?: number;
}
```

and so if I remove the `--no-semis`, I end up with the "official" style

```diff
 interface Props extends ViewProperties {
-  trigger1pxScrollHack?: boolean,
+  trigger1pxScrollHack?: boolean;
 }
```

So, FWIW - I'd want to have commas on all of them as I've had issues with typescript definition files due to this, and now have a linter in danger-js to warn me about it.